### PR TITLE
Documentation/zenko 2992 remove orbit

### PR DESCRIPTION
**Pull request by wabernat**
Removes Orbit UI and the term "Orbit" from all topics. 

Made XDM/Zenko split diagrams. Made changes to conf.py that enable XDM/Zenko split.

Removed extraneous junk as I encountered it.

### CODE REVIEW

File: docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/search_tool_characteristics.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Expression (PCRE) search syntax.
+
+Searchable Metadata and Tags
+----------------------------
+
+Because |product| supports two classes of custom tags, it is not possible to
+list all the ways to search data in |product|. At a mimimum, however, files will
+have the following fields inscribed, and these will be searchable:
+
+-  cache-control
```
Comment (miniscruff) [inline docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/search_tool_characteristics.rst:17]: hmm, the fact that some are kebab cased and some are camel-cased is a little lame.

File: docs/docsource/operation/index.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 |product| Operation
-===================
+====================
```
Comment (miniscruff) [inline docs/docsource/operation/index.rst]: was this intentional? It doesn't line up anymore.

**Review by miniscruff — APPROVED** [review]

File: docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/search_tool_characteristics.rst (hunk 1 above)
Comment (wabernat) [reply docs/docsource/operation/XDM_GUI/bucket_operations/metadata_search/search_tool_characteristics.rst:17]: Not my department... I just tell it like it is.

File: docs/docsource/operation/index.rst (hunk 2 above)
Comment (wabernat) [reply docs/docsource/operation/index.rst]: That extra character was for the typo I deleted!

**Review by miniscruff — APPROVED** [review]

**Review by lucieleonard — APPROVED** [review]

**Pull request merged**